test/suppose-process.test.js

```diff
@@ -143,7 +118,7 @@ describe('process', function()
     })
   })
 
-  it('should call a method as response', function(done)
+  it_posixOnly('should call a method as response', function(done)
```

Comment:
A better solution would be to use a command available for both systems, or add our own one written in Node.js.